.tekton/scripts/check-task-pipeline-bundle-repos.sh

```diff
@@ -22,7 +22,7 @@ echo "Checking existence of task and pipeline bundle repositories ..."
 
 # tasks
 for task_dir in $(find task/*/*/ -maxdepth 0 -type d); do
-    if [ ! -f $task_dir/kustomization.yaml ]; then
+    if [[ ! -f $task_dir/kustomization.yaml || ! -f $task_dir/recipe.yaml ]]; then
       task_name=$(oc apply --dry-run=client -f "$task_dir/*.yaml" -o jsonpath='{.metadata.name}')
     else
       task_name=$(oc apply --dry-run=client -k "$task_dir" -o jsonpath='{.metadata.name}')
```

Comment:
Not sure I understand the logic here

```
if the task directory has neither kustomization.yaml nor recipe.yaml:
    read task name directly from task yaml
else
    run kustomize and read task name from the result
```
I think the decision should still be based only on the existence of kustomization.yaml, no?

Reply:
Dropped this logic altogether, not sure if it made sense before, I think it might be leftover from a version I did when the generated files were not versioned...